### DISCUSSION TITLE
Continuation of recipe lines

### DISCUSSION THREAD

**Discussion by davised** [casey/just#1265] [Q&A] [answered]
Hi all,
I have some recipes that have a lot of parameters with defaults set, making the line very long, e.g. (there's more to the line still)

```
summarize-dada2 ASVTABLE=asvtable ASVTABLEV=asvtablev METADATA=metadata ...
```

Is there any plan for allowing continuation on the recipe lines? Or is this already possible and I just missed it?

Thanks for this software, it's really really cool.

Cheers,
Ed

**Comment by casey** [comment, marked as answer]
We could actually do it without requiring line continuations if the parser just ignored newlines when parsing recipe signatures. I created an issue to track it: #1266.

**Reply by davised** [reply]
Thanks Casey, really appreciate it!

**Comment by jraygauthier** [comment]
Should have been fixed via https://github.com/casey/just/pull/1551 since v1.15.
